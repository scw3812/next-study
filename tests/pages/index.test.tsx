import { render, screen } from '@testing-library/react';
import Home from '../../src/pages/index';

describe('Home', () => {
  it('renders a home text', () => {
    const { container } = render(<Home />);

    const homeTextElements = screen.getAllByText('Home');

    homeTextElements.forEach((el) => expect(el).toBeInTheDocument());

    expect(container).toMatchSnapshot();
  });
});
