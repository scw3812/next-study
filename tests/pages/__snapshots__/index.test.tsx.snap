// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a home text 1`] = `
<div>
  <div
    class="m-5 p-5 border border-solid border-gray-300"
  >
    <div>
      <button
        class="mr-4"
      >
        Home
      </button>
      <button
        class="mr-4"
      >
        About
      </button>
    </div>
    <p>
      Home
    </p>
  </div>
</div>
`;
